shopping-cart.html

```diff
@@ -104,6 +104,98 @@ <h2 class="shopping-cart__left-text">Mi carrito</h2>
+                      <path d="M9.45532 7.33333V11.3333" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                      <path d="M13.0046 7.33333V11.3333" stroke="#A6A6A6" stroke-width="2" stroke-linecap="round" stroke-linejoin="round" />
+                    </svg>Eliminar </a>
+                  <div class="purchase-summary__quantity-selector">
+                    <button class="purchase-summary__decrease"></button>
+                    <span class="purchase-summary__quantity">1</span>
+                    <button class="purchase-summary__increase"></button>
+                    <button class="purchase-summary__increase purchase-summary__increase--1"></button>
+                  </div>
+                </div>
+              </div>
+              <div class="purchase-summary__analytics-bord purchase-summary__analytics--bord-2"></div>
+            </div>
+          </div>
+        </div>
+        <div class="purchase-summary__payment">
+          <a href="#" class="purchase-summary__analytics__link"></a>
+          <div class="purchase-summary__payment-row">
+            <span class="purchase-summary__payment-code">Subtotal</span>
+            <h4 class="purchase-summary__payment-precio">$359,98</h4>
+          </div>
+          <div class="purchase-summary__payment-row">
+            <span class="purchase-summary__payment-code">IBTMS</span>
+            <h4 class="purchase-summary__payment-precio">$20.99</h4>
+          </div>
+          <div class="purchase-summary__payment-total">
+            <span class="purchase-summary__payment-quantity">Total</span>
+            <h1 class="purchase-summary__payment-precio purchase-summary__payment-precio--3">$380,97</h1>
+          </div>
+          <div class="purchase-summary__payment-shipping">
+            <img class="purchare-summary__payment-truck" src="image/Truck.png" alt="">
+            <p class="purchase-summary__payment-available">Disponible para envío gratis</p>
+          </div>
+          <a class="purchase-summary__payment-process">
+            <p class="purchase-summary__payment-text">PROCESAR EL PAGO </p>
```

Comment:
Los elementos de bloque no van dentro de elementos inline
```suggestion
            <span class="purchase-summary__payment-text">PROCESAR EL PAGO </span>
```